subsys/dfu/boot/targets/96b_nitrogen.h

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2017 Linaro
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief Bootloader device specific configuration.
+ */
+
+#define FLASH_DRIVER_NAME		CONFIG_SOC_FLASH_NRF5_DEV_NAME
+#define FLASH_ALIGN			4
+#define FLASH_AREA_IMAGE_0_OFFSET	0x08000
```

Comment:
The only define that should be needed from these files is the FLASH_DRIVER_NAME, and I think we should be figuring out how to get the DT code to generate the define based on the chosen node.